test/support/api/forms.js

```diff
@@ -1,62 +1,52 @@
 import _ from 'underscore';
-import { v4 as uuid } from 'uuid';
 import { getResource, mergeJsonApi } from 'helpers/json-api';
 
-import fxTestForms from 'fixtures/test/forms';
+import fxTestForms from 'fixtures/collections/forms';
 
-const TYPE = 'forms';
+const fxSampleForms = _.rest(fxTestForms, 1);
 
-export function getForm(data) {
-  const resource = getResource(_.sample(fxTestForms), TYPE);
+const TYPE = 'forms';
 
-  data = _.extend({ id: uuid() }, data);
+// Exporting only form needed for testing variance
+export const testForm = getResource(_.extend(fxTestForms[0], {
+  name: 'Test Form',
+}), TYPE);
 
-  return mergeJsonApi(resource, data);
+export function getForm(data) {
+  return mergeJsonApi(testForm, data);
 }
 
 export function getForms() {
-  return getResource(fxTestForms, TYPE);
+  return getResource(fxSampleForms, TYPE);
```

Comment:
I think this needs to be `[testForm, ...getResource(fxSampleForms, TYPE)],` and `routeForms` should use it.

Reply:
Updated 👍